### DISCUSSION TITLE
Управление функцией писка

### DISCUSSION THREAD

**Discussion by plesetsk** [TionAPI/HA-tion#97] [Ideas]
Может можно добавить функцию включения отключения писка бризера при передаче команд на включение? 
Это можно сделать через нативное приложение, что не всегда удобно.

Проблема в том, что когда ночью передается команда на включение от автоматизации этот писк может будить.

**Comment by IATkachenko** [comment]
Да, такая возможность есть: в модуле, вроде, все необходимое уже есть, так что большая часть работы уже сделана.
Завел #98

**Reply by plesetsk** [reply]
Спасибо!
